eq-author/src/components/CollapsibleToggled/index.js

```diff
@@ -28,22 +27,24 @@ const CollapsibleToggled = ({
   defaultOpen = false,
   headerContent,
   children,
+  onChange,
 }) => {
-  const [isOpen, toggleCollapsible] = useState(defaultOpen);
-
   return (
     <Wrapper data-test="CollapsibleToggled">
       <Header data-test="CollapsibleToggled__Header">
         <h2>{title}</h2>
         <ToggleSwitch
+          id={`${title}`}
           name={`${title}-toggle-switch`}
           hideLabels={false}
-          onChange={() => toggleCollapsible(!isOpen)}
-          checked={isOpen}
+          onChange={onChange}
+          checked={defaultOpen}
```

Comment:
```suggestion
          checked={isOpen}
```

If you do the change that renames this prop, this will need to change.